lib/ansible/runner.rb

```diff
@@ -430,12 +446,10 @@ def ansible_python_paths_raw(version)
         `python#{version} -c 'import site; print(":".join(site.getsitepackages()))'`.chomp
       end
 
-      def ansible_python_version
-        ansible_python_version_raw.match(/python version = (\d+\.\d+)\./)&.captures&.first
-      end
-
+      # NOTE: This method is ignored by brakeman in the config/brakeman.ignore
       def ansible_python_version_raw
-        `ansible --version 2>/dev/null`.chomp
+        ansible = venv_bin_path ? File.join(venv_bin_path, "ansible") : "ansible"
```

Comment:
Checking this allows it to run locally, on the old appliance where it doesn't live in the venv, and on the new appliance where is does live in the venv.